src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6168,10 +6173,13 @@ export class StudyViewPageStore {
                         : 0;
                     break;
                 }
-                case ChartTypeEnum.FUSION_GENES_TABLE: {
-                    count = this.molecularProfileSampleCountSet.result['fusion']
-                        ? this.molecularProfileSampleCountSet.result['fusion']
-                        : 0;
+                case ChartTypeEnum.STRUCTURAL_VARIANT_GENES_TABLE: {
+                    count =
+                        this.molecularProfileSampleCountSet.result[
+                            'structural_variants'
+                        ] ||
+                        this.molecularProfileSampleCountSet.result['fusion'] ||
```

Comment:
Not sure it's necessary but still want to point it out that we could have this `fusion` property as a constant.